lib/modules/manager/flux/readme.md

```diff
@@ -36,16 +36,16 @@ If `tag` _and_ `digest` exist then `digest` takes precedence.
 
 ### Flux system manifests support
 
-Support updating Flux system manifests generated during [Flux installation](https://fluxcd.io/docs/installation/#customize-flux-manifests).
+Renovate supports updating Flux system manifests generated during [Flux installation](https://fluxcd.io/docs/installation/#customize-flux-manifests). System manifests are identified via being named `gotk-components.y(a)ml` and containing comment headers as generated by `flux bootstrap`. Additional components are parsed and provided to `flux install` to generate a new manifest file.
```

Comment:
```suggestion
Renovate supports updating Flux system manifests generated during [Flux installation](https://fluxcd.io/docs/installation/#customize-flux-manifests).
System manifests are identified via being named `gotk-components.y(a)ml` and containing comment headers as generated by `flux bootstrap`.
Additional components are parsed and provided to `flux install` to generate a new manifest file.
```
one sentence per line

Reply:
```suggestion
Renovate can update Flux system manifests generated during [Flux installation](https://fluxcd.io/docs/installation/#customize-flux-manifests).
Flux system manifests files are `gotk-components.yml` or `gotk-components.yaml`.
These files also have comment headers generated by the `flux bootstrap` command.
Additional components are parsed and provided to `flux install` to generate a new manifest file.
```

I think this is better? This line still feels vague:

> Additional components are parsed and provided to `flux install` to generate a new manifest file.

We don't explain _which_ components are parsed and provided to `flux install`. If we add that explanation we probably make the text too verbose and in-depth. We should focus on the Renovate-specific parts, I think. So maybe it's best to drop that last line.

Reply:
I agree, it was leaning a little too much into implementation details. I've applied the suggested changes and removed the latter line about components.

On that though, is it worth popping a warning about [custom flags not being supported](https://github.com/renovatebot/renovate/issues/13952#issuecomment-1031620639) for things like `--cluster-name`? In such a case Renovate will overwrite these values with the defaults in its PR.